gbfs-validator/versions/schemas/v3.0-RC/manifest.json

```
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC/gbfs.md#manifestjson",
+  "description": "An index of gbfs.json URLs for each GBFS data set produced by a publisher. A single instance of this file should be published at a single stable URL, for example: https://example.com/gbfs/manifest.json.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description": "Last time the data in the feed was updated in POSIX time.",
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description": "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description": "GBFS version number to which the feed conforms, according to the versioning framework (added in v1.1).",
+      "type": "string",
+      "enum": [
+        "3.0-RC",
+        "3.0"
+      ]
+    },
+    "data": {
+      "type": "object",
+      "properties": {
+        "datasets": {
+          "description": "An array of objects, each containing the keys below.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "system_id": {
+                "description": "The system_id from system_information.json for the corresponding data set(s).",
+                "type": "string"
+              },
+              "versions": {
+                "description": "Contains one object, as defined below, for each of the available versions of a feed. The array MUST be sorted by increasing MAJOR and MINOR version number.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "version": {
+                      "description": "The semantic version of the feed in the form X.Y",
+                      "type": "string",
+                      "enum": [
+                        "1.0",
+                        "1.1",
+                        "2.0",
+                        "2.1",
+                        "2.2",
+                        "2.3",
+                        "3.0-RC"
```

I am waiting for a [confirmation](https://github.com/MobilityData/gbfs-json-schema/pull/88#issuecomment-1611730088) that v3.0 should also be added in the `datasets > versions > version` enum in `gbfs-json-schema/v3.0-RC/manifest.json`.

```suggestion
                        "3.0-RC",
                        "3.0"
```

---

I've re-imported the schemas from https://github.com/MobilityData/gbfs-json-schema, so the fix included in https://github.com/MobilityData/gbfs-json-schema/commit/5f12243d5735a608eef697212a627b9090eea31a is now present here.